src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -1,29 +1,32 @@
 package client
 
 import (
-	"fmt"
-	"io"
 	"log"
-	"net"
 	"net/http"
-	"net/http/httputil"
-	"strconv"
-	"sync"
+	"strings"
 	"time"
 
-	"github.com/getlantern/balancer"
+	"github.com/getlantern/flashlight/analytics"
+	"github.com/getlantern/flashlight/client"
+	"github.com/getlantern/flashlight/globals"
 	"github.com/getlantern/flashlight/util"
+	"github.com/getlantern/fronted"
 )
 
 const (
 	cloudConfigPollInterval = time.Second * 60
-	httpConnectMethod       = "CONNECT"
-	httpXFlashlightQOS      = "X-Flashlight-QOS"
 )
 
 // clientConfig holds global configuration settings for all clients.
 var clientConfig *config
 
+// MClient is an extension of flashlight client with a few custom declarations for mobile
+type MClient struct {
+	client.Client
+	hqfd   fronted.Dialer
```

Comment:
To be able to run ``StopClientProxy()`` with the sample program since flashlight client doesn't explicitly expose a stop method.